frontend/src/login/LoginFailedDialog.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  makeStyles
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  title: {
+    color: "#ef5350"
+  }
+}));
```

Comment:
Why did you choose to do `useStyles` instead of defining and using classes in `index.css`?

Reply:
Honestly, I was following an example on the material ui website. My arguments in favor of keeping it this way are
- `index.css` is already huge, making it hard to find stuff in right now
- having the CSS here makes it easy to reference directly from the code it affects and avoids having side effects on other pages
I would be okay with moving this stuff into a CSS file instead, but i would want it to be specific to the login page

Reply:
I think it would be better to have it in `index.css`. The reason it's like that in the Material UI documentation is to just show the user what styles are applied for that example specifically. You could add a comment in the CSS file to specify that they are for login. I think it's better if all styles can be found centralized in one file versus scattered around the project since some might be reused in the future for other components.